src/components/LmsConfigurations/index.jsx

```diff
@@ -74,6 +79,14 @@ class LmsConfigurations extends React.Component {
         />
       );
     }
+    if (blackboardConfig) {
+      return (
+        <BlackboardIntegrationConfigForm
+          config={camelCaseObject(blackboardConfig)}
+          enterpriseId={this.props.enterpriseId}
+        />
+      );
+    }
     return (
       <>
```

Comment:
This should likely be a div with the "container" class, or whatever class other pages use to get the correct margin / padding.

Reply:
Let's find a time to discuss in more detail. My guess is that this should probably be applied to Canvas and Moodle as well

Reply:
Agree, I think let's open a ticket for it, since multiple people are working on this.